dds_web/commands.py

```diff
@@ -775,75 +783,268 @@ def quarterly_usage():
+
+        # Save new rows
+        db.session.add_all(all_new_rows)
+        db.session.commit()
 
     except (sqlalchemy.exc.OperationalError, sqlalchemy.exc.SQLAlchemyError) as err:
-        flask.current_app.logger.exception(err)
         db.session.rollback()
+        flask.current_app.logger.error(
+            "Usage collection <failed> during step 2: calculating and saving usage. Sending email..."
+        )
+
+        # Send email about error
+        email_message: flask_mail.Message = flask_mail.Message(
+            subject=error_subject,
+            recipients=[email_recipient],
+            body=error_body,
+        )
+        send_email_with_retry(msg=email_message)
         raise
 
+    # 3. Send success email
+    flask.current_app.logger.info("Usage collection successful; Sending email.")
+    email_subject += " Usage records available for collection"
+    email_message: flask_mail.Message = flask_mail.Message(
+        subject=email_subject,
+        recipients=[email_recipient],
+        body=email_body,
+    )
+    send_email_with_retry(msg=email_message)
+
+
+@click.command("send-usage")
+@click.option("--months", type=click.IntRange(min=1, max=12), required=True)
+@flask.cli.with_appcontext
+def send_usage(months):
+    """Get unit storage usage for the last x months and send in email."""
+    # Imports
+    from dds_web.database import models
+    from dds_web.utils import current_time, page_query, send_email_with_retry
+
+    # Email settings
+    email_recipient: str = flask.current_app.config.get("MAIL_DDS")
+    # -- Success
+    email_subject: str = "[SEND-USAGE CRONJOB]"
+    email_body: str = f"Here is the usage for the last {months} months.\n"
+    # -- Failure
+    error_subject: str = f"{email_subject} <ERROR> Error in send-usage cronjob"
+    error_body: str = (
+        "There was an error in the cronjob 'send-usage', used for sending"
+        " information about the storage usage for each SciLifeLab unit. \n\n"
+        "What to do:\n"
+        "1. Check the logs on OpenSearch.\n"
+        "2. The DDS team should enter the backend container and run the command `flask send-usage`.\n"
+        "3. Check that you receive a new email indicating that the command was successful.\n"
+    )
+
+    end = current_time()
+    flask.current_app.logger.debug(f"Month now: {end.month}")
+
+    start = end - relativedelta(months=months)
+    flask.current_app.logger.debug(f"Month {months} months ago: {start.month}")
+
+    flask.current_app.logger.debug(f"Start: {start}")
+    flask.current_app.logger.debug(f"End: {end}")
+
+    # CSV files to send
+    csv_file_names = []
+
+    have_failed = False  # Flag to check if any csv files failed to be generated
+
+    # Iterate through units
+    for unit in models.Unit.query:
+        # Generate CSV file name
+        csv_file_name = pathlib.Path(
+            f"{unit.public_id}_Usage_Months-{start.month}-to-{end.month}.csv"
+        )
```

Comment:
I noticed this now: When I run this to collect the usage and send the csv for rows that have been collected for one month, the csv file is called e.g. `f"{unit.public_id}_Usage_Months-10-to-11.csv"` which may become confusing since it kind of seems like we're sending the usage for both months. 
Anyway, imo we shouldn't change this now, but I'll write this as info to Jonas and the others so that they know what the file name means and it's possible we'll need to change it in the future.